packages/cli/src/commands/deploy/index.ts

```diff
@@ -195,6 +195,16 @@ export default async (client: Client) => {
   let sourceFilesOutsideRootDirectory: boolean | undefined = true;
 
   if (status === 'not_linked') {
+    if (
+      localConfig &&
+      Object.prototype.hasOwnProperty.call(localConfig, 'projectSettings')
+    ) {
```

Comment:
```suggestion
    if (localConfig?.projectSettings) {
```

Reply:
Lol i forget that this syntax exists sometimes 😅